#!/usr/bin/python3
def raise_exception_msg(message=""):
    error_variable_name = "this variable does not exist " + message
    result = error_variable_name
    return result
